packages/actions/test/index.test.ts

```diff
@@ -63,7 +63,65 @@ describe("Sample e2e", () => {
         assert.isRejected(checkParticipantForCeremony({ ceremonyId: selectedCeremony.id }))
     })
 
-    afterEach(async () => {
+    it('should fail to create a sample ceremony without being a coordinator', async () => {
+        const ceremonyData = fakeCeremoniesData.fakeCeremonyOpenedDynamic
+        const circuitData = fakeCircuitsData.fakeCircuitSmallNoContributors
+        const ceremonyPrefix = ceremonyData.data.title.replace(/[`\s~!@#$%^&*()|+\-=?;:'",.<>\{\}\[\]\\\/]/gi, "-").toLowerCase()
```

Comment:
Could you explain this? Thanks!

Reply:
Nit: you could use the `ceremonyData.data.prefix` as a prefix for the ceremony. BTW, in this case you have implemented this correctly and gives back the same result, just a note for next test @ctrlc03

Reply:
Right I missed that property of the mock data, will change to clean up